aws-integration/README.md

```diff
@@ -0,0 +1,138 @@
+# Connect AWS X-Ray and CloudWatch to kyma
+
+## Overview 
+
+Sometimes you may want to transfer your logs, metrics and traces to other systems as well. Kyma provides a possibility to integrate it with some of the AWS components by deploying a custom OTEL Collector which would collect traces, logs, and metrics from kyma system and forward them to the AWS. In order to do this, we recommend you to use [AWS distribution](https://aws-otel.github.io) for OTEL Collector. 
```

Comment:
The Kyma Telemetry module supports you in integrating with observability backends in a convenient way. This example outlines how to integrate with [AWS CloudWatch](https://aws.amazon.com/cloudwatch/) as a backend. As CloudWatch is not supporting OTLP ingestion natively, it will require to deploy the [AWS Distro for OpenTelemetry](https://aws-otel.github.io) additionally.

Then add a diagram with the involved components, something like:
![aws integration drawio](https://github.com/kyma-project/examples/assets/7479371/ea0179ac-3d17-4742-9d43-86dd7f7bb76b)

Then structure the chapters around the diagram:
- Setup AWS and credentials
- Setup AWS distro
- Setup Kyma pipelines
- Deploy app